journal/original/32-epistles.adoc

```diff
@@ -1203,7 +1210,8 @@ My sincere love, in the unchangeable truth, is to you all, and my prayers to God
 [.signed-section-signature]
 William Edmundson
 
-=== Concerning Men`'s and Women`'s Meetings.
+[.letter-heading]
+Concerning Men`'s and Women`'s Meetings.
 
 [.salutation]
 Dear Friends, Brethren and Sisters;
```

Comment:
At the very end of this letter, where he says "This from your ancient friend and brother," ... lets put that on its own line under a [.signed-section-closing], like you have it in the modernized.